### PR TITLE
Added nav components include navlink

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: test/components/Layout.vue [hunk 1]
```diff
@@ -12,23 +12,23 @@
                   :fixed-drawer="fixedDrawer"
                   class="demo-layout-transparent">
         <mdl-layout-header-row title="Title here" slot="header">
-          <div class="mdl-layout-spacer"></div>
+          <mdl-layout-spacer/>
           <!-- Navigation -->
-          <nav class="mdl-navigation">
+          <mdl-nav>
             <a class="mdl-navigation__link" href="">Link</a>
```
Comment (pksunkara) [inline test/components/Layout.vue]: I don't know if I want to use `mdl-nav-link` here or not even though I added a separate test for it.

File: src/layout/spacer.vue [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<template>
+  <div class="mdl-layout-spacer"></div>
+</template>
```
Comment (posva) [inline src/layout/spacer.vue]: We should make this one a functional component as well

File: src/nav/nav.vue [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<template>
+  <nav class="mdl-navigation">
+    <slot></slot>
+  </nav>
+</template>
```
Comment (posva) [inline src/nav/nav.vue]: Should be a functional component as well

File: src/vue-mdl.js [hunk 4]
```diff
@@ -51,7 +54,10 @@ export const components = {
   MdlLayout,
   MdlLayoutHeaderRow,
   MdlLayoutDrawer,
-  MdlLayoutContent
+  MdlLayoutContent,
+  MdlLayoutSpacer,
+  MdlNav,
+  MdlNavLink
```
Comment (posva) [inline src/vue-mdl.js]: Can you add the components at the bottom too, please?

**Comment by pksunkara** [comment]
Done.

**Comment by posva** [comment]
Thanks!

**Comment by pksunkara** [comment]
Can you do a 1.0.3 release?

**Comment by posva** [comment]
Already done 🙂 It's a 1.1.0, though

**Comment by pksunkara** [comment]
@posva Do you have any idea why `@click` is not working on the navlink component? It is not working on the button component too for me.

**Comment by posva** [comment]
you need to use `.native`

**Comment by pksunkara** [comment]
@posva Adding `.native` fixed the button component. But navlink component is still not working.

**Comment by posva** [comment]
It should work, so make sure theres no wrapping element stopping the mouse
click propagation. If that's not it, then I'll love to have a minimal repro
in a jsfiddle.net so I can take a look

**Pull request merged**